src/components/profile/Profile.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import MissionJoin from '../mission/MissionJoin';
+import './Profile.css';
+import ProfileRocket from './ProfileRocket'; // eslint-disable-line import/extensions
```

Comment:
Hey @christianbiring1, Thank you for your suggestions. But we are not seeing any type of error in our machine. Before and after removing the eslint disable comment 
Eslint disable command remover :
![image](https://user-images.githubusercontent.com/16004612/233577441-3ae25d70-ca33-442b-8cad-da27b633e721.png)
Result 
 
![image](https://user-images.githubusercontent.com/16004612/233578563-2a8d8924-6a5b-4c27-9360-49a1e867f529.png)
Complied: 
![image](https://user-images.githubusercontent.com/16004612/233578868-2c3e07a9-0338-41d4-9f9c-0769be742867.png)

Reply:
@imranhossainemi 
I wasn't talking about linters disabled only, I was saying the path to your module, because if you try to deploy your app that way it surely gonna crash, as I have suggested you may consider putting all your components inside one directory named components instead of two as you have right now or you can name them different names